e2e/__tests__/multiProjectRunner.test.ts

```diff
@@ -513,14 +513,6 @@ describe("doesn't bleed module file extensions resolution with multiple workers"
 
     expect(configs).toHaveLength(2);
 
-    const [{name: name1}, {name: name2}] = configs;
```

Comment:
keep these, but just call them `id` instead of `name`

Reply:
I initially did this, however `id` doesn't exist. I'm confused and need some help understanding why `id` is not present